doc/dadbod.txt

```diff
@@ -126,8 +126,8 @@ MySQL ~
     mysql://[<user>[:<password>]@][<%2Fsocket%2Fpath>]/[<database>]
     mysql:[<database>]
 <
-Supported query parameters are `login-path`, `protocol`, `ssl-ca`, `ssl-cert`, and
-`ssl-key`, which pass the corresponding command line optins.
+Query parameters such as `login-path`, `protocol`, `ssl-ca`, `ssl-cert`, and
+`ssl-key` can be passed as the corresponding command line options.
```

Comment:
Instead of "can be", try "are".

Reply:
fixed
https://github.com/tpope/vim-dadbod/pull/109/commits/fb83d8325ba7a629c9a4dd46437ce1995268a091#diff-987e5fd0b9ceb853928de7a53b99bb36900a3fa56ec06fc6666691b487f3c1c5R130